.changeset/dull-walls-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@alephium/mobile-wallet": patch
+---
```

Comment:
Modal stays open after navigating to hidden assets screen


https://github.com/user-attachments/assets/6d7e0ef1-94d8-43ae-9052-91c0a8074908

Reply:
fixed in my PRs